test/services/enqueue_publishers_for_payout_service_test.rb

```diff
@@ -3,8 +3,14 @@
 require "test_helper"
 
 class EnqueuePublishersForPayoutServiceTest < NoTransactDBBleanupTest
+  include MockRewardsResponses
```

Comment:
At some point we should look into some test-suite-wide before blocks.  We're having to stub out the rewards responses in a million tests.